client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -99,11 +99,16 @@ public SparkShuffleManager(SparkConf conf, boolean isDriver) {
               + "use Celeborn as Remote Shuffle Service to avoid performance degradation.",
           SQLConf.LOCAL_SHUFFLE_READER_ENABLED().key());
     }
-    if (conf.getBoolean("spark.dynamicAllocation.shuffleTracking.enabled", false)) {
+    if ((Boolean) conf.get(package$.MODULE$.DYN_ALLOCATION_SHUFFLE_TRACKING_ENABLED())) {
```

Comment:
will enabling shuffle tracking cause degression even the ESS is enabled?

Reply:
Nope, Spark will check whether shuffle service enabled or not.

```scala
  private val shuffleTrackingEnabled = !conf.get(SHUFFLE_SERVICE_ENABLED) &&
    conf.get(DYN_ALLOCATION_SHUFFLE_TRACKING_ENABLED)
```

Reply:
such an unconditional warning looks incorrect then? especially when user want to use ESS as fallback

Reply:
Well, if in fallback case, both `local shuffle read` and `shuffle tracking` warn seems incorrect.

Maybe we should move those warn into registerShuffle methods to make sure there is not fallback case?

Reply:
after a second thought, I think it's fine. because we emphasis "... when use Celeborn as Remote Shuffle Service ..."

Reply:
what's the return type? why cast Boolean is required?

Reply:
Seems when we call `conf.get(package$.MODULE$.DYN_ALLOCATION_SHUFFLE_TRACKING_ENABLED())` in java may get Object type.

Reply:
that's interesting